src/katsdpmodels/band_mask.py

```diff
@@ -86,13 +86,11 @@ def is_masked(self, spectral_window: SpectralWindow, frequency: u.Quantity) -> A
     @classmethod
     def from_hdf5(cls: Type[_B], hdf5: h5py.File) -> _B:
         data = models.get_hdf5_dataset(hdf5, 'ranges')
-        if data.ndim != 1:
-            raise models.DataError(f'ranges dataset should have 1 dimension, found {data.ndim}')
         expected_dtype = np.dtype([
             ('min_fraction', 'f8'),
             ('max_fraction', 'f8'),
         ])
-        data = models.require_columns(data, expected_dtype)
+        data = models.require_columns('ranges dataset', data, expected_dtype, 1)
```

Comment:
Can we not synthesize the name from model_type + model_format to make the naming both more general and more consistent across different models ?

Reply:
Sorry, I don't understand the suggestion.

Reply:
:) Basically suggesting that the string 'ranges dataset' that is passed to produce decent error messages could be synthesized from the model_type and model_format of the class (in this case giving 'band_mask ranges'). Then we get some consistency across classes for the naming. No big deal though...

Reply:
Here 'ranges' is the name of an HDF5 dataset, and just happens to equal the model_format. Other model formats (e.g. bspline) use multiple HDF5 datasets (e.g. knots and coefficients).

Reply:
Fine - I guess I was trying to avoid multiple instances of the naming, since in line 88 you get_hdf5_dataset with a name, and then use something that looks very similar in the require_columns call.

Reply:
I'm with you now - have pushed a change to avoid repeating myself.